src/Qir/Runtime/stdlib/src/lib.rs

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#![deny(clippy::all, clippy::pedantic)]
```

Comment:
Are the other deny attributes needed given this at the crate level?

Reply:
I only just recently learned that I don't need to put it in every file :). I'll clean that up for clarity.